src/mc_tasks/ComplianceTask.cpp

```diff
@@ -22,6 +22,28 @@ std::function<double(double)> clamper(double value)
   return clamp;
 }
 
+const mc_rbdyn::ForceSensor & getSensor(const mc_rbdyn::Robots & robots,
+                                        unsigned int robotIndex,
+                                        const std::string & body)
+{
+  if(robotIndex >= robots.size())
+  {
+    LOG_ERROR_AND_THROW(std::runtime_error, "[mc_tasks::ComplianceTask] No robot with index "
+                                                << robotIndex << ", " << robots.size() << " robots loaded");
```

Comment:
We've already deprecated the use of robotIndex, wouln't it be better to access/log robot by name here instead?

Reply:
At this point there cannot be an error due to the name of the robot (this is handled by the robotToRobotIndex helper which gives an output related to the name).